examples/pipeline-templates-v1/configuration-inheritance/template.yml

```diff
@@ -0,0 +1,41 @@
+---
+schema: "1"
+id: configurationInheritance
+metadata:
+  name: Configuration Inheritance Example
+  description: A simple inheritance example of how to inherit parameters from a template
+
+configuration:
+  triggers:
+    - type: jenkins
+      job: "{{ jenkinsTriggerJob }}"
+      master: "{{ jenkinsMaster }}"
+      propertyFile: "{{ jenkinsPropertyFile }}"
+      enabled: true
+  parameters:
+    - name: baseOs
+      description: The baseOs to bake the image with
+  concurrentExecutions:
+    parraellel: true
```

Comment:
Typo here - but good news, you don't need `parallel` at all anymore - we removed this flag from Spinnaker entirely.